src/pretix/base/models/items.py

```diff
@@ -485,11 +492,21 @@ class Item(LoggedModel):
         null=True, blank=True,
         help_text=_('This product will not be sold before the given date.')
     )
+    available_from_mode = models.CharField(
+        choices=UNAVAIL_MODES,
+        default=UNAVAIL_MODE_HIDDEN,
+        max_length=16,
+    )
     available_until = models.DateTimeField(
         verbose_name=_("Available until"),
         null=True, blank=True,
         help_text=_('This product will not be sold after the given date.')
     )
+    available_until_mode = models.CharField(
+        choices=UNAVAIL_MODES,
+        default=UNAVAIL_MODE_HIDDEN,
+        max_length=16,
+    )
```

Comment:
These should also be exposed (and documented) in the API :)

Reply:
First time adding something to the API, could you have a look if it's correct? https://github.com/pretix/pretix/pull/3739/commits/c27c7292695ac862970557b3477c12b4307aee3e